src/main/java/fr/insalyon/creatis/moteur/plugins/workflowsdb/WorkflowsDBListener.java

```diff
@@ -121,9 +122,12 @@ public void executionCompleted(fr.cnrs.i3s.moteur2.execution.Workflow workflow,
             Workflow workflowBean = workflowDAO.get(workflowID);
 
             if (completed) {
-                workflowBean.setStatus(WorkflowStatus.Completed);
+                boolean outputProduced = didWorkflowProduceOutput();
+                workflowBean.setStatus(outputProduced ?
+                        WorkflowStatus.Completed : WorkflowStatus.Failed);
+                logger.print(TAG + "Execution finished with output : " + outputProduced);
             } else {
-                workflowBean.setStatus(WorkflowStatus.Killed);
+                workflowBean.setStatus(WorkflowStatus.Failed);
```

Comment:
I'm not sure about replacing the Killed status with Failed here

Reply:
Not an easy choice. In this case it means the workflow ended wrongly in moteur, because of an internal exception and not because of processor/job error. I'm not sure of all the possible causes but I think it is mostly due to gwendia/gasw transfer issues or malformed gwendia/gasw files. And think they should be qualified as Failed and not Killed status.

Reply:
OK, then let's go for Failed